bin/www

```diff
@@ -22,11 +22,11 @@ app.set('port', port);
 
 var server = http.createServer(app);
 
-models.sequelize.sync().then(function () {
-  server.listen(port);
-  server.on('error', onError);
-  server.on('listening', onListening);
-});
+// models.sequelize.sync().then(function () {
```

Comment:
Do you want to leave these code comments in here? Commented out code tends to get out of line with the working code really quickly - so it's usually a good idea to remove it - Git will hold the history for you.